### PR TITLE
Test topic code

### PR DESCRIPTION
**Pull request by gianlucb**
client app working

### CODE REVIEW

File: src/ServiceFabricPubSub/ServiceFabricPubSub.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26430.6
```
Comment (mcollier) [inline src/ServiceFabricPubSub/ServiceFabricPubSub.sln:4]: Why did the Visual Studio revision number go down? Shouldn't this be 15.0.26430.6? Granted, it is a very minor change according to https://www.visualstudio.com/en-us/news/releasenotes/vs2017-relnotes.

File: src/ServiceFabricPubSub/RequestRouter/Controllers/RequestController.cs [hunk 2]
```diff
@@ -21,127 +25,91 @@ public class RequestController : ApiController
...
+                HttpResponseMessage responseMessage = new HttpResponseMessage(HttpStatusCode.OK);
 
-            HttpResponseMessage topicResponseMessage;
-            using (HttpClient httpClient = new HttpClient())
-            {
-                HttpContent postContent = new StringContent(messageBody);
-                postContent.Headers.ContentType = new MediaTypeHeaderValue("application/json");
+                var msg = new PubSubMessage() { Message = message };
+                var topicSvc = ServiceProxy.Create<ITopicService>(new Uri($"fabric:/{tenantId}/{TenantApplicationTopicServiceName}/{topicName}"));
+                await topicSvc.Push(msg);
```
Comment (mcollier) [inline src/ServiceFabricPubSub/RequestRouter/Controllers/RequestController.cs]: Why change to use Service Fabric remoting instead of HTTP reverse proxy?

**Comment by mcollier** [comment]
@gianlucb I left a few comments / questions. Would you please take a look?

File: src/ServiceFabricPubSub/RequestRouter/Controllers/RequestController.cs (hunk 2 above)
Comment (gianlucb) [reply src/ServiceFabricPubSub/RequestRouter/Controllers/RequestController.cs]: I had few issues with port conflicts. I can try again to put back the http listener

File: src/ServiceFabricPubSub/Admin/Admin.csproj [hunk 3]
```diff
@@ -20,6 +20,9 @@
     <PackageReference Include="Microsoft.ServiceFabric.Data" Version="2.6.204" />
```
Comment (brentstineman) [inline src/ServiceFabricPubSub/Admin/Admin.csproj:20]: Please don't hesitate to ping the group for help on the port conflict issue. Its part of the upside of having a team to help solve problems. :)

**Comment by gianlucb** [comment]
I put back the Reverse Proxy logic.
it is working now! the only thing seems that the first message in a queue is lost. After the second everything works. Probably need some debug on the message duplication method

**Pull request merged**